aya/src/programs/mod.rs

```diff
@@ -206,6 +206,10 @@ pub enum ProgramError {
         /// program name
         name: String,
     },
+
+    /// TraceFS not found.
+    #[error("tracefs non found")]
+    TraceFsNotFound,
```

Comment:
I think that this is too specific - it's not something that will likely ever be handled programmatically so it doesn't deserve its own variant. We should probably add a `ProgramError::IOError(#[from] IOError)` and create a `Error::new(ErrorKind::Other, "tracefs not found")` error for this. Then move some of the other fringe variants into it (not as part of this PR, but at some point in the future)

Reply:
sounds good